tests/manager/api/test_axis.py

```diff
@@ -551,6 +551,12 @@ def test_patron_activity(self, axis360: Axis360Fixture):
         assert isinstance(hold2, HoldInfo)
         assert isinstance(loan, LoanInfo)
 
+        # If the activity includes something with a Blio format, it is not included in the results.
+        data = axis360.sample_data("availability_with_axisnow_fulfillment.xml")
+        axis360.api.queue_response(200, content=data)
+        results = axis360.api.patron_activity(patron, "pin")
+        assert len(results) == 0
```

Comment:
I'm a little confused by this one. The sample data looks like it has `ePub`, `Blio`, and `AxisNow` formats available. I thought we'd be able to use the `ePub` format, so I think I'm missing something here.

```xml
  ...
    <availableFormats>
      <formatName>Blio</formatName>
      <formatName>ePub</formatName>
      <formatName>AxisNow</formatName>
    </availableFormats>
  ...
```

Reply:
`availableFormats` is a bit misleading, those are the formats the item is available in before its checked out. Once its checked out, its format is locked to the format in `checkoutFormat` element in the response. So in this case `<checkoutFormat>Blio</checkoutFormat>`.

I'll add a clarifying comment to the code.